tests/test_index.cpp

```diff
@@ -253,5 +254,52 @@ TestIndex::TestSerializeFile(const IndexPtr& index_from,
         }
     }
 }
+void
+TestIndex::TestConcurrentKnnSearch(const TestIndex::IndexPtr& index,
+                                   const TestDatasetPtr& dataset,
+                                   const std::string& search_param,
+                                   float recall,
```

Comment:
rename recall -> expected_recall

Reply:
rename in another pr for all test recall